docs/self-hosted/cli.md

```diff
@@ -22,16 +22,10 @@ For server-side CLI, all functionality can be accessed by running `npx directus
 ### Initialize a New Project
 
 ```bash
-npx directus init
+npm init directus-project <project-folder>
 ```
```

Comment:
Do we want to provide the command for other PMs as well? E.g. similar to https://vitepress.dev/guide/getting-started#setup-wizard

Reply:
Not sure if thats appropriate for this tiny snippet 🤔 I think that would belong in a broader effort to improve non-docker self-hosted guides.

Reply:
Makes sense 👍